compiler/src/dotty/tools/dotc/tasty/TastyImpl.scala

```diff
@@ -24,6 +25,15 @@ object TastyImpl extends scala.tasty.Tasty {
     def toTasty(implicit ctx: Context): TypeTree = PickledQuotes.quotedTypeToTree(x)
   }
 
+  // ===== Show =====================================================
+
+  def defaultShow: Show[this.type] = showExtractors
```

Comment:
Using `def` here makes the member unstable problems, see c9dbecce233978048cedad6a0154e1df85d1f83a — I'm not using the API you used, but I'd expect both to work as they usually do.